sqlitecluster/SQLiteNode.cpp

```diff
@@ -1174,7 +1155,6 @@ bool SQLiteNode::update() {
         if (STimeNow() > _stateTimeout) {
             // Give up
             SHMMM("Timed out waiting for SUBSCRIPTION_APPROVED, reconnecting to leader and re-SEARCHING.");
-            unique_lock<shared_mutex> leadPeerLock(_stateMutex);
```

Comment:
Why remove this?

Reply:
We lock once at the sop of `update` now: https://github.com/Expensify/Bedrock/pull/1252/files#diff-2d1c5f24a4b7ebc3076630d8ee3a1155ad5e7789abe72cff977e615541c319d6R536